### ISSUE TITLE
Installing nixops from nixpkgs on nixos 21.11 fails

### ISSUE THREAD

**Issue by wykurz** [NixOS/nixops#1491]
Running `nix-env -iA nixos.nixops` results in:

```
...
running egg_info
writing ruamel.base.egg-info/PKG-INFO
writing namespace_packages to ruamel.base.egg-info/namespace_packages.txt
writing top-level names to ruamel.base.egg-info/top_level.txt
writing dependency_links to ruamel.base.egg-info/dependency_links.txt
reading manifest file 'ruamel.base.egg-info/SOURCES.txt'
writing manifest file 'ruamel.base.egg-info/SOURCES.txt'
Copying ruamel.base.egg-info to build/bdist.linux-x86_64/wheel/ruamel.base-1.0.0-py2.7.egg-info
Installing build/bdist.linux-x86_64/wheel/ruamel.base-1.0.0-py2.7-nspkg.pth
running install_scripts
builder for '/nix/store/wckbqd6zsbjb9iapyqvpyc9qyk1wpbsy-python2.7-ruamel-yaml-clib-0.2.4.drv' failed with exit code 1
cannot build derivation '/nix/store/rksn65k8pz69j2jbb3cdd39x8pw3rfwg-python2.7-ruamel-yaml-0.16.13.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/9w5ld179l8w0iyff57ws3flszpph1fyy-python2.7-construct-2.10.54.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/r18kc25n6ak41zix5yrld688r5lgwa59-python2.7-brotlipy-0.7.0.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/sqnfrh6apyrdy2cnkrhan4i16n7cirrn-python2.7-httpbin-0.7.0.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/83knpz4yzpbjlg698bcs33yhkjczq84c-python2.7-pytest-httpbin-1.0.0.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/50k88ikb7k805q7ls9i8pls2mwa8r7hp-python2.7-vcrpy-3.0.0.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/xn2qqm6hpmbnrdqxx29jgxf3kqvdsra9-python2.7-datadog-0.42.0.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/zshvfjp2hqrybhsyc0lg6dzm2mdmy4ds-nixops-1.7.drv': 1 dependencies couldn't be built
error: build of '/nix/store/zshvfjp2hqrybhsyc0lg6dzm2mdmy4ds-nixops-1.7.drv' failed
```

**Comment by Tchekda** [comment]
Did you try with `nixopsUnstable` ? Because it's the only one to "work" on my side with `21.11`

**Comment by wykurz** [comment]
@Tchekda yes, it installs fine but then I run into the issue you reported: https://github.com/NixOS/nixops/issues/1490 🤷🏻

**Comment by wmertens** [comment]
Could we backport just the fixes to make stable work on 21.11?

**Comment by roberth** [comment]
```
$ nix-build -I nixpkgs=channel:nixos-21.11 -A nixops
/nix/store/p7hfw29c36qi5ab7ky6sqkpykcj5imlx-nixops-1.7
```

If it doesn't build on another `system`, open an issue on `nixpkgs` instead.
